File: 0389-find-the-difference/0389-find-the-difference.cpp
class Solution {
public:
    char findTheDifference(string s, string t) {
        int sum=0;
        int sum2=0;
        for(auto it:s){
            sum+=it;
        }
        for(auto it:t){
            sum2+=it;
        }
        return char(sum2-sum);
    }
};

File: 0389-find-the-difference/0389-find-the-difference.go
func findTheDifference(s string, t string) byte {
    sum1 :=0
    var sum2 int =0
    for _, char :=range s{
        sum1+=int(char)
    }
    for _, char:=range t{
        sum2+=int(char)
    }
    return byte(sum2-sum1)
}